src/provider/hyrax_pc.rs

```diff
@@ -319,51 +310,42 @@ where
   G: Group<CE = HyraxCommitmentEngine<G>>,
 {
   type CE = G::CE;
-  type ProverKey = HyraxProverKey<G>;
+  type ProverKey = ();
   type VerifierKey = HyraxVerifierKey<G>;
   type EvaluationArgument = HyraxEvaluationArgument<G>;
 
   #[tracing::instrument(skip_all, name = "HyraxEvaluationEngine::setup")]
   fn setup(
     ck: &<<G as Group>::CE as CommitmentEngineTrait<G>>::CommitmentKey,
   ) -> (Self::ProverKey, Self::VerifierKey) {
-    let span = tracing::span!(tracing::Level::INFO, "hyrax PK");
-    let _guard = span.enter();
-    let pk = HyraxProverKey::<G> {
-      ck_s: G::CE::setup(b"hyrax", 1),
-    };
-    drop(_guard); 
-    drop(span); 
+    let pk = ();
 
     let span = tracing::span!(tracing::Level::INFO, "hyrax VK");
     let _guard = span.enter();
-    let vk = HyraxVerifierKey::<G> {
-      ck_v: ck.clone(),
-      ck_s: G::CE::setup(b"hyrax", 1),
-    };
-    drop(_guard); 
-    drop(span); 
+    let vk = HyraxVerifierKey::<G> { ck_v: ck.clone() };
+    drop(_guard);
+    drop(span);
 
     (pk, vk)
   }
 
   #[tracing::instrument(skip_all, name = "HyraxEvaluationEngine::prove")]
   fn prove(
-    ck: &CommitmentKey<G>,
-    pk: &Self::ProverKey,
+    _ck: &CommitmentKey<G>,
+    _pk: &Self::ProverKey,
     transcript: &mut G::TE,
     comm: &Commitment<G>,
     poly: &[G::Scalar],
     point: &[G::Scalar],
-    eval: &G::Scalar,
+    _eval: &G::Scalar,
   ) -> Result<Self::EvaluationArgument, SpartanError> {
     transcript.absorb(b"poly_com", comm);
 
     let span = tracing::span!(tracing::Level::INFO, "poly_m_construct");
     let _guard = span.enter();
     let poly_m = MultilinearPolynomial::<G::Scalar>::new(poly.to_vec());
```

Comment:
`.to_vec()` -> `.bound()` allocates excessively.

Probably fine given these are only sqrt size?

Reply:
will address in a follow-up PR